### ISSUE TITLE
we should have `-Wstrict-prototypes`

### ISSUE THREAD

**Issue by fenugrec** [candle-usb/candleLight_fw#103]
and/or enable an optional cppcheck  pass at compile-time

**Comment by ReXT3D** [comment]
FWIW, the current codebase produces only one warning (per board) with `-Wstrict-prototypes` enabled:

```
/home/pi/candleLight_fw/src/interrupts.c:63:1: warning: function declaration isn't a prototype [-Wstrict-prototypes]
 typedef void(*pFunc)();
 ^~~~~~~
```

**Comment by fenugrec** [comment]
Closed via PR #105 . Thanks !
